app/services/exchange_rates/update_currency_rates_service.rb

```diff
@@ -11,7 +12,7 @@ def call
 
       ExchangeRateCurrencyRate.db.transaction do
         rates = build_rates(response)
-        included_rates = ExchangeRateCountryCurrency.distinct(:currency_code).select_map(:currency_code)
+        included_rates = @type == ExchangeRateCurrencyRate::MONTHLY_RATE_TYPE ? ExchangeRateCountryCurrency.distinct(:currency_code).select_map(:currency_code) : ExchangeRateCountryCurrency::SPOT_RATE_CURRENCY_CODES
```

Comment:
With regards to the monthly rates shouldnt we be filtering based by the live country rates rater than just all the ones in the table?

Reply:
Yeah i realise that but its defo something that needs to be changed soon